src/sparql/cl-xrefs.sparql

```diff
@@ -0,0 +1,9 @@
+prefix oio: <http://www.geneontology.org/formats/oboInOwl#>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+
+SELECT ?cls ?xref WHERE 
+{
+  ?cls a owl:Class ; 
+     oio:hasDbXref ?xref .
+    FILTER(isIRI(?cls) && (STRSTARTS(str(?cls), "http://purl.obolibrary.org/obo/CP_") || STRSTARTS(str(?cls), "http://purl.obolibrary.org/obo/CL_") || STRSTARTS(str(?cls), "http://purl.obolibrary.org/obo/cl#")))
```

Comment:
Sure you want to include deprecated classes in the report?

Reply:
Thanks for the reminder. I'll change the query.

Reply:
done, thanks